tests/spoke/cluster_info_test.go

```diff
@@ -78,13 +79,24 @@ prefixes:
 - 192.168.0.0/16
 - 10.96.0.0/12`)
 
+			DeferCleanup(func() {
+				Expect(k8sClient.Delete(ctx, node)).Should(Succeed())
```

Comment:
Can we wait until the object is completely deleted .

Reply:
yeah we can , but we are not fetching the node again. We just want to trigger the node reconciler